### PR TITLE
Rewrite to move to AWS Lambda

### LINKED ISSUE #56: Only send the bee emoji once
This message is send twice, which is confusing, because once should be enough for users to show they are joining for dinner.

### PR DESCRIPTION
**Pull request by pingiun**
Closes #56 #45

### Summary
- Replaced pugsql with AWS DynamoDB invocations.
- Rewrote certain parts for AWS lambda things
- Added GitHub action to deploy on Lambda
- Removed `requests` dependency

### How to test
We'll wait to see if it works on Wednesday?

### TODO:
- [x] Use database to map wbw to slack users
- [x] Write Slack slash command to add users to database
- [x] Change deploy action to only deploy on master

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -21,11 +21,7 @@
         - name: Setup Poetry
```
Comment (joren485) [inline .github/workflows/test.yaml]: This still says "Poetry"

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          with:
+            python-version: '3.8'
+
+        - name: Install pytz
+          run: |
+            python -m pip install --upgrade --no-cache-dir pip
+            python -m pip install --no-cache-dir --target . pytz
+
+        - name: Zip the function
+          run: |
```
Comment (joren485) [inline .github/workflows/deploy.yaml]: This does not have to be a multiline item.

File: .github/workflows/deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        - name: Deploy poll bot
+          uses: appleboy/lambda-action@master
+          with:
+            aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws_region: ${{ secrets.AWS_REGION }}
+            function_name: what-are-we-eating-today
+            zip_file: deployment.zip
+
+        - name: Deploy slash command
```
Comment (joren485) [inline .github/workflows/deploy.yaml:38]: Please add some documentation on how to use this.

**Review by joren485 — CHANGES_REQUESTED** [review]
The README still contains info about `poetry`.

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**